docs/group_desc_en.md

```diff
@@ -1,20 +1,20 @@
 `Zencoders` is a group of software developers and informatics enthusiasts
```

Comment:
I started from 2014 italian description. Probably it would be better to merge the first two phrases like that:

> Zencoders is a group of software developers and informatics enthusiasts leveraging knowledge sharing amongst its members to realize common projects and develop innovating ideas.